cmd/context/context.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package context
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/analytics"
+	okContext "github.com/okteto/okteto/pkg/cmd/context"
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/okteto/okteto/pkg/okteto"
+	"github.com/spf13/cobra"
+)
+
+type ContextOptions struct {
+	Token       string
+	Namespace   string
+	clusterType okteto.ClusterType
+}
+
+// Context points okteto to a cluster.
+func Context() *cobra.Command {
+	ctxOptions := &ContextOptions{}
+	cmd := &cobra.Command{
+		Use:     "context [url|k8s-context]",
+		Aliases: []string{"ctx"},
+		Args:    utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#context"),
+		Short:   "Points okteto to a cluster",
+		Long: `Points okteto to a cluster
```

Comment:
I think `Points okteto to a cluster` may be too vague. I think the vagueness may be due to the overloaded nature of the command. 

What does it mean to point to a cluster? If I don't have kubectl configuration setup then I need that first. If I do then I just need to use the "name" from the kubectl configuration, which may have been change out of band?

Reply:
This is the definition of a kubernetes context, which mostly apply to an okteto context too:
```
A kubernetes context is just a set of access parameters that contains a Kubernetes cluster, a user, and a namespace. kubernetes Context is essentially the configuration that you use to access a particular cluster & namespace with a user account.
```
I would go with something like:
```
Short: "Manage your okteto context"
Long: "Manage your okteto context

Explain what is an okteto context. Explain how it works when it is based on a kubernetes context, and explain how it works when it is based on an Okteto cluster
```